app/models/role.rb

```diff
@@ -1,9 +1,25 @@
 class Role < ActiveRecord::Base
-  has_and_belongs_to_many :users
   belongs_to :resource, polymorphic: true
+  has_and_belongs_to_many :users
 
   scopify
 
-  LABELS = ['Attendee', 'Volunteer', 'Speaker', 'Sponsor', 'Press', 'Keynote Speaker']
```

Comment:
Roles will only be about actionable roles and we will not have labels mixed with roles.